app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListFragment.kt

```diff
@@ -84,6 +88,17 @@ class TransactionListFragment : SafeOverviewBaseFragment<FragmentTransactionList
                             adapter.submitData(PagingData.empty())
                         }
                     }
+                    is ShowError -> {
+                        binding.progress.visible(false)
+                        when(viewAction.error) {
+                            is Offline -> {
+                                snackbar(requireView(), R.string.error_no_internet)
+                            }
+                            else -> {
+                                //TODO: handle error
```

Comment:
TODO?

Reply:
at this point we should show empty placeholder instead of content